common/logging.py

```diff
@@ -28,6 +31,38 @@ def format(self, record):
             fmt_str = fmt_str.replace('\n', '') + '|'
         return fmt_str
 
+
+def setup_cw_logging(logger):
+    """Setup CloudWatch logging"""
+    key_id = os.environ.get('CW_AWS_ACCESS_KEY_ID')
+    secret = os.environ.get('CW_AWS_SECRET_ACCESS_KEY')
+    if not (key_id and secret):
+        logger.info('CloudWatch logging disabled due to missing access key')
+        return
+
+    try:
+        with open('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'r') as namespace_file:
```

Comment:
I think, you can use `with` -> `except` without using `try`, because `with` already includes `try`-like behaviour.

Reply:
this is not true:
```
with open("/tmp/a.txt", "rt") as my_file:
    print(my_file.read())
except:
    print("error")
```

```
  File "/tmp/a.py", line 3
    except:
         ^
SyntaxError: invalid syntax
```

Reply:
Hm, ok. I was wrong.